### ISSUE TITLE
Valid value returned in case of an invalid input

### ISSUE THREAD

**Issue by susuter** [vaadin/vaadin-time-picker#97]
We are using the vaadin-time-picker as well as the vaadin-date-picker in our application. We observed a different behavior of the two components when an invalid input such as "asdf" is entered. While the vaadin-date-picker returns an empty value (empty string), the vaadin-time-picker returns the previously entered valid time. In our application, this results in saving the previously selected (or default) time without any notification to the user.

In short: in our application we use the vaadin-time-picker and the vaadin-date-picker to select a date time properties, which is then saved. We have a MVC setup and use html files as view. We use customized encoders for both fields. The retrieved strings in the encoders are different.

We suggest to review the return values in the vaadin-time-picker such that it returns null or an empty value when an invalid input such as "asdf" is entered.
